### DISCUSSION TITLE
Different permissions for readOne and readMany?

### DISCUSSION THREAD

**Discussion by cemreinanc** [directus/directus#6365] [Feature Requests]
Is there a permission difference between reading a single item and being able to see the item list?
We wanted to restrict querying and listing on some collections with public role but still giving access to read items by their keys.

We just had a use case for this and solved quickly with a custom api endpoint. But I think this could be helpful for many people when we have this option in permissions.

**Comment by philleepflorence** [comment]
It looks like you are referring to [BREAD (browse, read, edit, add, delete) and CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) - so a separate Browse and Read (Item) permission?
Mind sharing your implementation, I have similar use cases.

**Comment by benhaynes** [comment]
I agree that we'll want to add a new permission option for Collections. We can either call this `browse` or `list`, and it can be added to the end of the existing CRUD options. We'll need to align on which of the following this new permission controls:

* Collection Detail (Item Listing Pages)
* Relational Selection (Item Listing Drawers)
* API Endpoint (`/items`, but not `/items/PK`)
* Core/System Modules (eg: Files, Users, Insights)
    * Files and Users make sense because there's one collection per module. However Insights has multiple, Docs has none, and Settings is managed via the `admin` flag. So these might be better abstracted to a new Role config set.

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
